ConnorAndersonLarson/PlanetParty
# Feature/planet detail

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ X ] Feature
- [ X ] Refactor

### Are there any known bugs?

- [ X ] No

### Summary of the changes
Create a single Planet view component and add basic styling. Also refactor API call to get data from a different API and add functions to clean data.


### What ticket(s) do the changes resolve?
Single planet view and fetch API data. Solves CORS error we were getting with old API.

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
The planet info fact is hard coded so that data will not change when sort by type.


### How to Test Changes?
Pull down branch, make sure state is holding API info. No CORS errors of response errors in the console.


### Next Iterations

- Pass planet fact to each card when correct sort button is clicked
- Allow a user to click on a card, that will then display single planet view with respective facts
- Do we need to make a single API call for when a planet is clicked?
